app/app_mobile.go

```diff
@@ -1,22 +1,22 @@
+//go:build !ci && (android || ios)
 // +build !ci
-
-// +build android ios mobile
```

Comment:
Why have you removed the mobile build flag? it is super helpful for testing mobile driver on a desktop computer.

Reply:
This is a little bit tricky here. Testing a mobile driver on a desktop that relying on a different implementation is actually less useful. On desktop, we are currently depending on OpenGL 3.2 whereas mobile depends on ES. The application lifetime also manages differently. I agree that the current facility of testing with a mobile compatible size, but most of the runtime issues actually never detected. Hence I removed it. It would be more useful if we test directly on a mobile simulator.

Reply:
This was useful to test apps using the mobile driver. It reported as `true` if you `fyne.CurrentDevice().IsMobile()` and other features for example.
The render pipeline may indeed be GL instead of GLES - but graphics is not set for mobile devices specifically - painting is separate from the driver anyway.
If in the future you think we should remove functionality, please don't bury the change in a refactor PR!